pulpcore/pulpcore/app/settings.py

```diff
@@ -60,10 +60,13 @@
     'pulpcore.app',
 ]
 
-# add plugins to INSTALLED_APPS after pulpcore
+# Enumerate the installed Pulp plugins during the loading process for use in the status API
+INSTALLED_PULP_PLUGINS = []
+
 for entry_point in iter_entry_points('pulpcore.plugin'):
     plugin_app_config = entry_point.load()
     INSTALLED_APPS.append(plugin_app_config)
+    INSTALLED_PULP_PLUGINS.append(plugin_app_config.split('.')[0])
```

Comment:
`plugin_app_config.split('.')[0]` seems unnecessary

Reply:
so I've played a bit with the `pkg_resources` module and I guess this would better read:
```python
for entry_point in iter_entry_points('pulpcore.plugin'):
    pulp_app_config = entry_point.load()
    INSTALLED_APPS.append(plugin_app_config)
    INSTALLED_PULP_PLUGINS.append(entry_point.module_name)
```